example/index.html

```diff
@@ -38,6 +38,8 @@
   <script src="https://cdnjs.cloudflare.com/ajax/libs/OverlappingMarkerSpiderfier-Leaflet/0.2.6/oms.min.js"></script>
 
   <script src="lib/glify.js"></script>
+
+  <script src="https://kit.fontawesome.com/92fa0312fd.js" crossorigin="anonymous"></script>
```

Comment:
Can we download this font-awesome file and add that to LEL repo in /images folder so that the button image is there even if we are offline?

Reply:
And then remove this script also. Thanks!

Reply:
We could include it using NPM and then it'd be in `node_modules`?